components/organisms/ChangePasswordUser/ChangePasswordUser.tsx

```diff
@@ -107,14 +68,37 @@ export const ChangePasswordUser = () => {
       );
     }
   };
+  const validatePasswordUser = async () => {
```

Comment:
This variable should be `validateUserPassword`

Reply:
changed